nautobot/virtualization/templates/virtualization/virtualmachine.html

```diff
@@ -11,8 +11,15 @@
 {% endblock extra_buttons %}
 
 {% block nav_tabs %}
-        <li role="presentation"{% if not active_tab %} class="active"{% endif %}>
-            <a href="{{ object.get_absolute_url }}">Virtual Machine</a>
+        <li role="presentation"{% if not active_tab and not request.GET.tab or request.GET.tab == "main" %} class="active"{% endif %}>
+            <a href="{{ object.get_absolute_url }}#main" onclick="switch_tab(this.href)" aria-controls="main" role="tab" data-toggle="tab">
+                Virtual Machine
+            </a>
+        </li>
+        <li role="presentation"{% if request.GET.tab == 'advanced' %} class="active"{% endif %}>
+            <a href="{{ object.get_absolute_url }}#advanced" onclick="switch_tab(this.href)" aria-controls="advanced" role="tab" data-toggle="tab">
+                Advanced
+            </a>
```

Comment:
It would be better to change this so that it overrides block `extra_nav_tabs` with the additional tabs only, rather than repeating the contents of the base `nav_tabs` block.